libnmstate/nm/bridge.py

```diff
@@ -110,9 +110,9 @@ def get_info(nmdev):
     info[LB.CONFIG_SUBTREE] = {
         LB.PORT_SUBTREE: _get_bridge_ports_info(port_profiles),
         LB.OPTIONS_SUBTREE: {
-            LB.MAC_AGEING_TIME: bridge_setting.props.ageing_time,
-            LB.GROUP_FORWARD_MASK: bridge_setting.props.group_forward_mask,
-            LB.MULTICAST_SNOOPING: bridge_setting.props.multicast_snooping,
+            'mac-ageing-time': bridge_setting.props.ageing_time,
+            'group-forward-mask': bridge_setting.props.group_forward_mask,
+            'multicast-snooping': bridge_setting.props.multicast_snooping,
```

Comment:
I replaced the schema constants with the strings because if I use the constants the line is over 80 characters and linter complains but black does not complain. At the same time if I split the lines then black complains but linter does not complain.

Any suggestion? Thanks!

Reply:
```python
props = bridge_setting.props
LB.Options.MULTICAST_SNOOPING: props.multicast_snooping
```

Reply:
Done! Thanks :)